pkg/config/agent/config.go

```diff
@@ -192,6 +192,14 @@ type AntreaProxyConfig struct {
 	// Services will not be load-balanced). Values can be a valid ClusterIP (e.g. 10.11.1.2) or a Service name
 	// with Namespace (e.g. kube-system/kube-dns)
 	SkipServices []string `yaml:"skipServices,omitempty"`
+	// When ProxyExternalIPs is set to false, AntreaProxy no longer load-balances traffic destined to the
+	// ExternalIPs of LoadBalancer Services. This is useful when the external LoadBalancer provides additional
+	// capabilities (e.g. TLS termination) and it is desirable for Pod-to-ExternalIP traffic to be sent to the
```

Comment:
My concern is that: for externalTrafficPolicy of a  LoadBalancer Service is Cluster, which means the source IP address of client should be reserved, for the connection whose source is a Pod, and destination is a ExternalIP(Pod-to-ExternalIP traffic mode). I'm a little confused about how the reply packets go back to external LB to guarantee the symmetry of for a connection. 

- Request packet path:  Pod -> Antrea gateway -> external LB -> Antrea gateway -> Endpoint
- Reply packet path:  Endpoint -> Antrea gateway(without SNAT, I think the reply packet will be routed to the Pod since the source IP address of the client is reserved).

Reply:
> for externalTrafficPolicy of a LoadBalancer Service is Cluster, which means the source IP address of client should be reserved

Do you mean when `externalTrafficPolicy` is **Local**, in which case the IP is preserved?

The external Load Balancer must SNAT the traffic, which is typically the case or can be configured (for AVI for example it is the default), in order for the reply traffic to go back through the external LB. That applies regardless of the `externalTrafficPolicy`.

Reply:
Probably @hongliangl means the case LB does not SNAT, e.g. LB is inline and embedded in the gateway? That is quite uncommon though.

Reply:
> The external Load Balancer must SNAT the traffic, which is typically the case or can be configured (for AVI for example it is the default), in order for the reply traffic to go back through the external LB. That applies regardless of the externalTrafficPolicy.

If SNAT is performed regardless of the `externalTrafficPolicy`, it makes sense.